bobtemplates/plone/base.py

```diff
@@ -78,21 +78,32 @@ def is_string_in_file(configurator, file_path, match_str):
 
 def update_file(configurator, file_path, match_str, insert_str):
     """Insert insert_str into given file, by match_str."""
+    changed = False
+
     with open(file_path, 'r+') as xml_file:
         contents = xml_file.readlines()
         if match_str in contents[-1]:  # Handle last line, prev. IndexError
             contents.append(insert_str)
+            changed = True
         else:
             for index, line in enumerate(contents):
                 if (
                     match_str in line and
                     insert_str not in contents[index + 1]
                 ):
                     contents.insert(index + 1, insert_str)
+                    changed = True
                     break
         xml_file.seek(0)
         xml_file.writelines(contents)
 
+    if not changed:
+        print(
+            "WARNING: We couldn't find the match_str, "  # NOQA
```

Comment:
Please no general ``# NOQA`` tagging, please add case, for example: ``# NOQA: E501``